protocol/mqtt_paho/v2/context.go

```diff
@@ -0,0 +1,37 @@
+package mqtt_paho
+
+import "context"
+
+// ProtocolContext allows a Receiver to understand the context of a request.
+type ProtocolContext struct {
+	Topic    string
+	QoS      byte
+	Retained bool
+}
```

Comment:
would we also consider to support `packet id` and `packet properties`?

https://mosquitto.org/man/mosquitto-8.html

Reply:
Thanks @skeeey! Append these two parameters.